README.md

```diff
@@ -337,6 +339,15 @@ Installs [AzCopy](https://github.com/Azure/azure-storage-azcopy)
 * USE_azcopy: Enable this feature
 * DEBUG_azcopy: Debug this feature
 
+### bitwarden
+
+Installs the [Bitwarden cli](https://bitwarden.com/help/cli/)
```

Comment:
just cosmetics: i prefer cli in capital letters

Reply:
Fixed